packages/fluxible-addons-react/connectToStores.js

```diff
@@ -59,6 +59,7 @@ function createComponent(Component, stores, getStateFromStores, customContextTyp
     });
 
     hoistNonReactStatics(StoreConnector, Component);
+    StoreConnector.WrappedComponent = Component;
```

Comment:
In our other Fluxible related components we have used the lowercase `wrappedComponent` so we may want to be consistent here. I'm curious what other libraries are using in case there is a standard.